partner_email_check/__init__.py

```diff
@@ -1,2 +1,3 @@
 # -*- coding: utf-8 -*-
 from . import models
+from . import tests
```

Comment:
You don't have to import tests here. It's a special folder that is automatically imported during unit testing only.